lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -248,6 +248,7 @@ def site_map(record_location: str) -> str:
     """
     location_site_map = {
         'greek': 'UWGreek',
+        'PICAWA': 'PICAWA',
```

Comment:
I didn't see this value added into the redcap project yet (just childcare and greek), but as long as the value stays PICAWA, based on my read of the mapping, it would be just fine to deploy this even before its added in redcap 👍

Reply:
Yep, the new site is still in the drafted changes (thanks for double-checking!). Izzy says the enrollments are happening tomorrow, so it may be best to deploy this before the REDCap changes are in production so that the ETL doesn't break.